espresso.py

```diff
@@ -70,20 +73,25 @@ def retrieve_restaurant(rest_id):
         logging.error(f'Exception was thrown: {str(ex)}')
         ret_val = {'success': False,
                    'id': rest_id,
+                   'restaurant': None,
                    'message': f'Server failure: restaurant with id number {rest_id} could not be retrieved',
+                   'api_version': API_VERSION
                    }
         if isinstance(ex, sqlalchemy_exc.ProgrammingError):
             http_status = 500
         elif isinstance(ex, sqlalchemy_exc.DataError):
             http_status = 400
     else:
         if not rest:
-            ret_val = {'success': False, 'message': f'No restaurant with id {rest_id} found'}
+            ret_val = {'success': False, 'id': rest_id, 'restaurant': None,
+                       'message': f'No restaurant with id {rest_id} found',
+                       'api_version': API_VERSION
```

Comment:
would making this value an integer make sense?

Reply:
Maybe a float value would be better, then you could have, e.g.  0.5  or  1.2